generators/server/templates/gradle/docker.gradle.ejs

```diff
@@ -18,30 +18,27 @@
 -%>
 buildscript {
     repositories {
-        jcenter()
+      gradlePluginPortal()
     }
-
     dependencies {
-        classpath 'com.bmuschko:gradle-docker-plugin:3.5.0'
+      classpath "gradle.plugin.com.google.cloud.tools:jib-gradle-plugin:0.9.10"
     }
 }
 
-apply plugin: com.bmuschko.gradle.docker.DockerRemoteApiPlugin
-
-import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+apply plugin: com.google.cloud.tools.jib.gradle.JibPlugin
 
-task copyDockerFiles(type: Copy) {
-    description = "Copy Dockerfile and required data to build directory"
-    from 'src/main/docker'
-    from "${project.buildDir}/libs"
-    into { "${project.buildDir}/docker" }
-    include "*"
-    exclude "**/*.yml"
+jib {
+    from {
+        image = 'openjdk:8-jre-alpine'
```

Comment:
there is a constant for the base image

Reply:
hum... I must say I'm not sure extracting the image and port values in `gradle.properties` worth it.
1- They are already consider as constant in the actual file and it does not looks like a limitation
2- It forces to work on 2 files to define a docker build
3- for consistency sake, it we do it for `from.image` and `ports` why not for the other constants like `jvmFlag` or even the target image name

Reply:
@danielpetisme You need to use the generator constants like docker base image, server port, etc. @pascalgrimaud does not mean extracting those values to a property file just using the right EJS templating.

Reply:
Ok make sense. Will do
Sorry for the misunderstanding.

Reply:
I was commenting with my phone and wasn't clear enough sorry
See https://github.com/jhipster/generator-jhipster/blob/master/generators/server/templates/src/main/docker/Dockerfile.ejs#L19
We should use `image = '<%= DOCKER_JAVA_JRE %>'` instead